cpp/FileTransferAgent.cpp

```diff
@@ -300,7 +300,8 @@ void Snowflake::Client::FileTransferAgent::compressSourceFile(
 {
   CXX_LOG_DEBUG("Starting file compression");
   
-  char tempDir[100];
+  char tempDir[4096]={0};
+  memset(tempDir, 0, 4096);
```

Comment:
The previous line should already set the array to 0 and there should be no need to memset again.

Reply:
True. Removed the later.